lib/bolt/config.rb

```diff
@@ -64,7 +64,7 @@ def self.from_file(configfile, overrides = {})
              end
 
       data = load_defaults(project).push(
-        filepath: project.config_file,
+        filepath: configfile,
```

Comment:
If you pass `--configfile /path/to/bolt.yaml`, we will actually load `/path/to/bolt-project.yaml` if it exists. Will this cause us to instead report that we loaded `/path/to/bolt.yaml` when we didn't?

Reply:
I don't think so, that logging happens above on line 62 and is only if the configfile *isn't* bolt-project.yaml:
```
lucy ~/githubs/bolt (GH-2116) $ be bolt plan show --configfile project/bolt.yaml --log-level debug
Analytics opt-out is set, analytics will be disabled
Loaded project from '/home/lucy/githubs/bolt/project'
bolt-project.yaml contains valid config keys, bolt.yaml will be ignored
Loaded configuration from /home/lucy/.puppetlabs/etc/bolt/bolt-defaults.yaml
Project-level configuration in bolt.yaml is deprecated if using bolt-project.yaml. Transport config should be set in inventory.yaml, all other config should be set in bolt-project.yaml.
Loading modules from /home/lucy/githubs/bolt/bolt-modules:/home/lucy/githubs/bolt/project/modules:/home/lucy/githubs/bolt/modules:/home/lucy/githubs/bolt/project/spec/fixtures/modules:/home/lucy/githubs/bolt/project/spec/fixtures/apply:/home/lucy/githubs/bolt/modules
aggregate::count
aggregate::nodes
aggregate::targets
canary
facts
facts::external
facts::info
puppet_agent::run
puppetdb_fact
reboot
secure_env_vars
terraform::apply
terraform::destroy
```

Reply:
This is kind of hacky way to make this *slightly* more correct but not entirely. The `project.config_file` will still be wrong, but we just don't actually use it anywhere so it doesn't matter. Since we're removing it soon I'm not inclined to do the restructuring needed to make it correct.

Reply:
Ah right I forgot we got rid of the `config_loaded` method, which was using this. :thumbsup: